src/app/constants/sorts.js

```diff
@@ -1,8 +1,9 @@
-import allowedFilters from './filters';
-
-const sorts = {
-  filter: allowedFilters,
-  order: ['asc', 'desc'],
+export const sortMap = {
+  Relevance: [],
+  'Title A-Z': [{ field: 'title', dir: 'ASC' }],
```

Comment:
Dash is missing in render (checked Firefox and Chrome)

![Screen Shot 2019-06-07 at 10 01 03 AM](https://user-images.githubusercontent.com/720703/59109984-17000180-890c-11e9-8467-8a5cfaa28589.png)

Reply:
Solved

Reply:
Confirmed that its working for me